data-reconciliation-app/src/endpoints/all.ts

```diff
@@ -1,2 +1,3 @@
 export * from "./ingest";
```

Comment:
nit: do we want to change the original file to ingest-json.ts?!?!

Reply:
IMHO we can keep it as it is given that it's our default model, and we are providing an extra feature to allow CSV.

Dropping this for now, we can revisit it if you disagree :)